File: R/leibniz.r
# Rscript leibniz.r

k = 1
s = 0
for(i in 0:99999) {
	if (i%%2 == 0){
		s = s+ 4/k
	}else{
		s = s- 4/k
	}
	k=k+2
}
print(s)

File: Java/leibniz.java
class leibniz{
	public static void main(String args[]){
		int d = 1;
		double s = 0;
		for(int i = 0;i<100000;i++){
			if(i%2==0)
				s += 4./d;
			else
				s -= 4./d;
			d+=2;
		}
		System.out.println(s);
	}
}